src/components/magnet/magnet-canvas.tsx

```diff
@@ -34,13 +36,15 @@ interface IState {
   rotating2?: boolean;
 }
 
+const initialY = 300;
+
```

Comment:
Latest version has this moved up to about y=260 to avoid having coil magnet images overlap with controls.